homeassistant/components/kulersky/strings.json

```diff
@@ -0,0 +1,20 @@
+{
+  "title": "Kuler Sky",
```

Comment:
No need for this, HA defaults to the manifest title. Since it's a brandname, we don't want it translated.